binr/rax2/rax2.c

```diff
@@ -75,6 +75,24 @@ static void print_ascii_table() {
 	printf("%s", ret_ascii_table());
 }
 
+static int escape_to_dec(int i, char *str) {
+	switch (str[i]) {
+		case 'n': return 10; break;
```

Comment:
case must be indented at the same level as the switch and there's no need to break if you return